### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by guenk**
Adding a Navbar with styles and burger menu.

### CODE REVIEW

**Review by Sax593 — COMMENTED** [review]
The font and logo images are not used and react router is installed but not used

**Review by Sax593 — DISMISSED** [review]
Good job

File: frontend/src/components/Myspace/Myspace.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import './Style.scss';
+
+export default function Myhome() {
+
+
+
+
+
+
```
Comment (LoicBrassart) [inline frontend/src/components/Myspace/Myspace.jsx]: Where's my Prettier ? è_é

File: frontend/src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
-    <p></p>
+
+    <Myhome />
+
+    
+
+
+
+
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/Home.jsx]: Where !?
୧( ಠ Д ಠ )୨

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils, plz fix'em!

File: frontend/src/components/Myspace/Style.scss [hunk 3]
```diff
@@ -1,122 +1,120 @@
+html {
+  overflow-y: hidden;
+}
+main {
+  overflow-y: hidden;
+}
+main {
+  overflow-y: hidden;
+}
```
Comment (Sax593) [inline frontend/src/components/Myspace/Style.scss]: Death Code

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Sax593 — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Ongoing conflicts, plz fix

**Review by Sax593 — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**